### DISCUSSION TITLE
How can i uninstall shairport-sync and all its dependencies?

### DISCUSSION THREAD

**Discussion by socksprox** [mikebrady/shairport-sync#1534] [Q&A]
How?

**Comment by mikebrady** [comment]
You can disable and remove Shairport Sync as follows:

1. Disable the startup script -- `# systemctl disable shairport-sync`
2. Delete the app `shairport-sync` -- use `$ which shairport-sync` to find it.
3. Disable the NQPTP startup script -- `# systemctl disable nqptp`.
4. Delete the app `nqptp` -- use `$ which nqptp` to find it.
5. Delete the files `/etc/shairport-sync.conf` and `/etc/shairport-sync.conf.sample`

This will still leave a few insert script files, but they are quite harmless.

The system libraries that are needed for Shairport Sync will remain installed.
